src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/Api/UnitTestingRemoteHostClientWrapper.cs

```diff
@@ -19,9 +19,9 @@ internal UnitTestingRemoteHostClientWrapper(RemoteHostClient underlyingObject)
 
         internal RemoteHostClient UnderlyingObject { get; }
 
-        public async Task<UnitTestingKeepAliveSessionWrapper> TryCreateUnitTestingKeepAliveSessionWrapperAsync(string serviceName, CancellationToken cancellationToken)
+        public async Task<UnitTestingKeepAliveSessionWrapper> TryCreateUnitTestingKeepAliveSessionWrapperAsync(string serviceName, object callbackTarget, CancellationToken cancellationToken)
```

Comment:
📝 This signature change may require a coordinated insertion if it's currently in use.